tests/testdata/PE/x64/msvc-15.0-foo-bar.dll.txt

```diff
@@ -0,0 +1,10 @@
+    FILE SIZE        VM SIZE    
+ --------------  -------------- 

```

Comment:
This file is 1:1 bloaty output on windows, I left that in to test the 'Normalizing'.